test/intl402/Temporal/PlainMonthDay/prototype/monthCode/chinese-calendar-dates.js

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plainmonthday.prototype.monthCode
+description: monthCode should work for Chinese calendar leap dates
+features: [Temporal]
+includes: [temporalHelpers.js]
+---*/
+
+const calendar = "chinese";
+
+const leapMonthCases = [
+  { year: 1971, month: 6, monthCode: "M05L", day: 1, referenceYear: 1971 },
+  { year: 1974, month: 5, monthCode: "M04L", day: 1, referenceYear: 1963 },
+  { year: 1976, month: 9, monthCode: "M08L", day: 1, referenceYear: 1957 },
+  { year: 1979, month: 7, monthCode: "M06L", day: 1, referenceYear: 1960 },
+  { year: 1982, month: 5, monthCode: "M04L", day: 1, referenceYear: 1963 },
+  { year: 1984, month: 11, monthCode: "M10L", day: 1, referenceYear: 1870 },
+  { year: 1987, month: 8, monthCode: "M07L", day: 1, referenceYear: 1968 },
+  { year: 1990, month: 6, monthCode: "M05L", day: 1, referenceYear: 1971 },
+  { year: 1993, month: 4, monthCode: "M03L", day: 1, referenceYear: 1966 },
+  { year: 1995, month: 9, monthCode: "M08L", day: 1, referenceYear: 1957 },
+  { year: 1998, month: 6, monthCode: "M05L", day: 1, referenceYear: 1971 },
+  { year: 2001, month: 5, monthCode: "M04L", day: 1, referenceYear: 1963 },
+  { year: 2004, month: 3, monthCode: "M02L", day: 1, referenceYear: 1947 },
+  { year: 2006, month: 8, monthCode: "M07L", day: 1, referenceYear: 1968 },
+  { year: 2009, month: 6, monthCode: "M05L", day: 1, referenceYear: 1971 },
+  { year: 2012, month: 5, monthCode: "M04L", day: 1, referenceYear: 1963 },
+  { year: 2014, month: 10, monthCode: "M09L", day: 1, referenceYear: 1832 },
+  { year: 2017, month: 7, monthCode: "M06L", day: 1, referenceYear: 1960 },
+  { year: 2020, month: 5, monthCode: "M04L", day: 1, referenceYear: 1963 },
+  { year: 2023, month: 3, monthCode: "M02L", day: 1, referenceYear: 1947 },
+  { year: 2025, month: 7, monthCode: "M06L", day: 1, referenceYear: 1960 },
+  { year: 2028, month: 6, monthCode: "M05L", day: 1, referenceYear: 1971 },
+  { year: 2031, month: 4, monthCode: "M03L", day: 1, referenceYear: 1966 },
+  { year: 2033, month: 12, monthCode: "M11L", day: 1, referenceYear: 1813 },
```

Comment:
This referenceYear is out of range, we should delete this case until https://github.com/tc39/proposal-intl-era-monthcode/issues/60 is resolved.